### PR TITLE
Updating README.md

### PR DESCRIPTION
**Pull request by RGKarthik**

### CODE REVIEW

File: sdk/agrifood/azure-agrifood-farming/README.md [hunk 1]
```diff
@@ -245,15 +271,25 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
...
+[azure_subscription]: https://azure.microsoft.com/free/
+[change_log]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/agrifood/azure-agrifood-farming/CHANGELOG.md
 [cla]: https://cla.microsoft.com
-[code_of_conduct]: https://opensource.microsoft.com/codeofconduct/
-[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
+[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
+[code_of_conduct]: https://opensource.microsoft.com/codeofconduct/
+[default_azure_credential]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity#defaultazurecredential/
+[farm_hierarchy]: https://aka.ms/FarmBeatsFarmHierarchyDocs
```
Comment (ramya-rao-a) [inline sdk/agrifood/azure-agrifood-farming/README.md:285]: Lets have all the aka.ms links point to a single page that is already live. Once you have the individual docs up, you can go change the aka.ms link

File: sdk/agrifood/azure-agrifood-farming/README.md [hunk 2]
```diff
@@ -245,15 +271,25 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
...
+[farm_operations_docs]: https://aka.ms/FarmBeatsFarmOperationsDocumentation
+[install_farmbeats]: https://aka.ms/FarmBeatsInstallDocumentationPaaS
+[product_docs]: https://docs.microsoft.com/azure/azure-agrifood-farming/
+[pip]: https://pypi.org/project/pip/
+[pypi]: https://pypi.org/project/azure-agrifood-farming/
```
Comment (ramya-rao-a) [inline sdk/agrifood/azure-agrifood-farming/README.md:290]: We can skip having a link for this for the first preview. We can add it back for the next one

File: sdk/agrifood/azure-agrifood-farming/README.md [hunk 3]
```diff
@@ -44,6 +53,20 @@ client = FarmBeatsClient(endpoint="https://<my-account-name>.farmbeats.azure.net
 ```
 
 ## Key concepts
+Basic understanding of below terms will help to get started with FarmBeats client library.
+
+* [Farm Hierarchy][farm_hierarchy]: Is a collection of below entities.
```
Comment (iscai-msft) [inline sdk/agrifood/azure-agrifood-farming/README.md:58]: Generally for key concepts, we make each key concept an h3 level header. See [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/README.md#key-concepts) for an example.

I would recommend making an h3 for `Farm Hierarchy`, `Scenes`, and `Farm Operations`

File: sdk/agrifood/azure-agrifood-farming/README.md [hunk 4]
```diff
@@ -245,15 +271,25 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
...
-[code_of_conduct]: https://opensource.microsoft.com/codeofconduct/
-[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
+[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
+[code_of_conduct]: https://opensource.microsoft.com/codeofconduct/
+[default_azure_credential]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity#defaultazurecredential/
+[farm_hierarchy]: https://aka.ms/FarmBeatsFarmHierarchyDocs
+[farm_operations_docs]: https://aka.ms/FarmBeatsFarmOperationsDocumentation
+[install_farmbeats]: https://aka.ms/FarmBeatsInstallDocumentationPaaS
+[product_docs]: https://docs.microsoft.com/en-us/azure/industry/agriculture/overview-azure-farmbeats
```
Comment (ramya-rao-a) [inline sdk/agrifood/azure-agrifood-farming/README.md:288]: Locales in links are discouraged to allow the locales of the user to take precedence
```suggestion
[product_docs]: https://docs.microsoft.com/azure/industry/agriculture/overview-azure-farmbeats
```

File: sdk/agrifood/azure-agrifood-farming/README.md (hunk 2 above)
Comment (RGKarthik) [reply sdk/agrifood/azure-agrifood-farming/README.md]: Done

File: sdk/agrifood/azure-agrifood-farming/README.md (hunk 4 above)
Comment (RGKarthik) [reply sdk/agrifood/azure-agrifood-farming/README.md]: Fixed

**Review by ramya-rao-a — APPROVED** [review]

File: sdk/agrifood/azure-agrifood-farming/README.md (hunk 3 above)
Comment (RGKarthik) [reply sdk/agrifood/azure-agrifood-farming/README.md]: Fixed

**Pull request merged**